libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/dashboardDnD.spec.ts

```diff
@@ -0,0 +1,116 @@
+// (C) 2022 GoodData Corporation
+
+import * as Navigation from "../../tools/navigation";
+import { EditMode } from "../../tools/editMode";
+import { AttributeFilter, FilterBar } from "../../tools/filterBar";
+import { DashboardHeader } from "../../tools/dashboardHeader";
+import { DashboardMenu } from "../../tools/dashboardMenu";
+import { WidgetDropZone } from "../../tools/enum/DropZone";
+import { Widget } from "../../tools/widget";
+import { Dashboard } from "../../tools/dashboards";
+import { WidgetConfiguration } from "../../tools/widgetConfiguration";
+
+Cypress.Cookies.defaults({
+    preserve: ["GDCAuthTT", "GDCAuthSTT", "_csrfToken"],
+});
+
+Cypress.on("uncaught:exception", (error) => {
```

Comment:
As I mentioned before, we should move to index.ts in the support folder to use for all tests.

Reply:
not all tests need this

Reply:
ex:  tests in 02-sdk-ui-filters, 03-sdk-ui-charts

Reply:
I don't think we have any problems when we move it to `index.ts` although the others don't use it

Reply:
will update code for this in a follow-up ticket